docs/release_notes/release_notes.md

```diff
@@ -9,26 +9,91 @@ has_children: true
 # Release Notes
 
 
-## Version 5.0.0 (September 2024)
+## Version 5.0.0 (October 2024)
 
 * ### Alert Categorization
+    * Categorized the alerts into SOC and Compliance teams.
+    * Added **Team** filter on Overview dashboard.
+    * Added team configuration for SOC and Compliance teams under **Cyences Settings > Cyences App Configuration > Cyences Alerts Configuration** section.
+    * Updated the alerting logic (for regular and digest alerts) based on the teams.
 
+* ### BlockShield Integration
+    <!-- TODO - Add detailed info on this section. -->
+    * Removed everything related to honeyDB, blocked IPs.
+
+* ### F5 BIGIP
+    * Added new dashboard named **F5 BIGIP ASM**.
+    * Added new alert named **F5 BIGIP - Not Blocked Attacks**.
+
+* Added new alerts for the Sophos Firewall:
+    * Sophos Firewall - Lost Connection to Sophos Central
+    * Sophos Firewall - VPN Tunnel Down
+    * Sophos Firewall - Gateway Down
+    * Sophos Firewall - Advanced Threat Detected
+
+* Added new alerts for the MSSQL and Oracle databases:
+    * MSSQL - Database Changes
+    * MSSQL - Role Changes
+    * Oracle - Database Changes
+    * Oracle - Role Changes
+
+* Added new alerts for the Defender ATP:
+    * Defender ATP - System is Offboarded
+    * Defender ATP - System is not Connected since a Week
+
+* Added "Object Type" and "Object Name" filter on **MSSQL** and **Oracle** dashboards.
 
 * Removed the following deprecated alerts:
     * O365 - Login Failure Outside Home Country Due To Multi Factor Authentication
     * O365 - Successful Login Outside Home Country
     * Authentication - Successful VPN Login Outside Home Country
     * Linux - Change in Sudo Access of Local Linux Account
 
-* Removed the [Splunk Add-on for RWI - Executive Dashboard](https://splunkbase.splunk.com/app/5063/) dependency.
+* Removed the following other app dependent macros and defined related macro in app itself:
```

Comment:
No need to mention this point to release notes actually.
For now move it at the end in a section called **### For Splunk Admins**